pytket/phir/phirgen.py

```diff
@@ -200,11 +203,14 @@ def append_cmd(cmd: tk.Command, ops: list[dict[str, Any]]) -> None:
             ops.append(op)
 
 
-def genphir(inp: list[tuple[Ordering, Layer, Cost]]) -> str:
+def genphir(
+    inp: list[tuple[Ordering, Layer, Cost]], *, machine_ops: bool = True
```

Comment:
I wasn't aware that one can use `*` alone like that. But on another note, I wonder if `machine_ops` should be `False` by default.

Reply:
I had heard of it, but never used it until it was suggested by ruff at https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/

I used `True` as the default because that's the more common case. But happy to hear an argument in favor of `False`.